chain-10-tree-1.json

```diff
@@ -393,37 +425,50 @@
         "responsibilities": [
           {
             "link": "",
-            "label": "The Hats Active Contributor hat inherits the responsibilities of the Hats Community Member hat"
+            "label": "Follow Community Agreements",
+            "description": "The Hats Active Contributor hat inherits the responsibilities of the Hats Community Member hat"
           },
           {
             "link": "",
-            "label": "Track and participate actively in the Hats Community Telegram channel"
+            "label": "Participate in shared communications channels",
+            "description": "Track and participate actively in the Hats Community Telegram channel"
           },
           {
             "link": "",
-            "label": "Participate in Town Halls when possible and review the notes if/when you cannot make it"
+            "label": "Participate in Town Halls",
+            "description": "Participate in Town Halls when possible and review the notes if/when you cannot make it"
           },
           {
             "link": "https://app.charmverse.io/hats-protocol/page-0225506388762724",
-            "label": "Record contributions as appropriate using the Hats contribution tracking system"
+            "label": "Record contributions",
+            "description": "Record contributions as appropriate using the Hats contribution tracking system"
           }
         ],
         "authorities": [
           {
             "link": "",
```

Comment:
link to the [contribution instructions](https://app.charmverse.io/hats-protocol/how-to-submit-a-contribution-for-protorep-consideration-23301472932733813)